### PR TITLE
Added Ukrainian Language

### PR DESCRIPTION
**Pull request by dmytro-ielkin**
Hi,

I have added Ukrainian resources for datetime and timespan humanization as well as ordinazer, number to words converter. I don't have ReSharper so I hope there aren't any issues with that.

If there are feel free to let me know.

I'm using Humanizer in production website and looking forward to using updated nugget package when ready.

Regards,
Dmytro IELKIN

### CODE REVIEW

File: src/Humanizer.Tests/TimeSpanHumanizeTests.cs [hunk 1]
```diff
@@ -82,8 +82,8 @@ public void TimeSpanWithMaxTimeUnit(int ms, string expected, TimeUnit maxUnit)
         }
 
         [Theory]
-        [InlineData(0, 3, "no time")]
```
Comment (hazzik) [inline src/Humanizer.Tests/TimeSpanHumanizeTests.cs]: Any reasons why you have changed "no time" to "just now"? I'm asking because this can be used with general time-spans, which are not bound to any particular momentum.

Comment (dmytro-ielkin) [inline src/Humanizer.Tests/TimeSpanHumanizeTests.cs]: This was intended for cases when for example a post is just added and time span is 0 milliseconds so it would make sense to show 'just now'

But for what I understand from your comment sometimes users just want to translate such a time span to 'no time'

Would you like me to revert it back?

Comment (hazzik) [inline src/Humanizer.Tests/TimeSpanHumanizeTests.cs]: Yes please revert these changes. We can open separate issue for "no time"/"just now" and discuss there.

**Comment by dmytro-ielkin** [comment]
Resources and tests are reverted to 'no time'

File: src/Humanizer/Localisation/Formatters/UkrainianFormatter.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
+﻿using Humanizer.Localisation.GrammaticalNumber;
+
+namespace Humanizer.Localisation.Formatters
+{
+    internal class UkrainianFormatter : DefaultFormatter
+    {
+        public UkrainianFormatter()
+            : base("uk-UA")
```
Comment (hazzik) [inline src/Humanizer/Localisation/Formatters/UkrainianFormatter.cs]: Can it be specified as "uk"?

**Comment by dmytro-ielkin** [comment]
I tried but in that case it loads default resources

**Comment by mexx** [comment]
You should also rename `uk-UA.resx` file as well, then it should work, look at `ru` as example.

**Comment by dmytro-ielkin** [comment]
I see. Will do so.

**Comment by dmytro-ielkin** [comment]
Locale code updated

**Comment by dmytro-ielkin** [comment]
When do you think this can be merged and released in a new version on nuget? I'm asking because we want to use it in production.

File: src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt [hunk 3]
```diff
@@ -245,6 +245,10 @@ public class In
 
 public class InflectorExtensions
 {
+    public void AddIrregular(string singular, string plural) { }
```
Comment (mexx) [inline src/Humanizer.Tests/ApiApprover/PublicApiApprovalTest.approve_public_api.approved.txt]: I think something gone wrong with the rebase, as this methods are part of another PR.

File: src/Humanizer/Properties/Resources.resx [hunk 4]
```diff
@@ -462,4 +462,4 @@
   <data name="TimeSpanHumanize_MultipleWeeks_TrialQuadral" xml:space="preserve">
     <value>{0} weeks</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```
Comment (mexx) [inline src/Humanizer/Properties/Resources.resx]: Please revert this change, as it's not adding any value.

File: src/Humanizer/Properties/Resources.ru.resx [hunk 5]
```diff
@@ -339,4 +339,4 @@
   <data name="DateHumanize_SingleYearFromNow" xml:space="preserve">
     <value>через год</value>
   </data>
-</root>
\ No newline at end of file
+</root>
```
Comment (mexx) [inline src/Humanizer/Properties/Resources.ru.resx]: Same here, please revert this change, as it's not adding any value.

File: src/Humanizer/InflectorExtensions.cs [hunk 6]
```diff
@@ -144,23 +144,42 @@ public string Apply(string word)
             }
         }
 
-        private static void AddIrregular(string singular, string plural)
+        /// <summary>
```
Comment (mexx) [inline src/Humanizer/InflectorExtensions.cs]: Again changes in this file are part of another PR.

**Comment by mexx** [comment]
@hazzik Besides the comments, LGTM

**Comment by dmytro-ielkin** [comment]
Waiting for feedback. @mexx @hazzik

File: src/Humanizer.Tests/Localisation/ru-RU/TimeSpanHumanizeTests.cs [hunk 7]
```diff
@@ -124,4 +124,4 @@ public void NoTime()
             Assert.Equal("нет времени", TimeSpan.Zero.Humanize());
         }
     }
-}
\ No newline at end of file
+}
```
Comment (hazzik) [inline src/Humanizer.Tests/Localisation/ru-RU/TimeSpanHumanizeTests.cs]: Please revert this

**Comment by hazzik** [comment]
@dmytro-ielkin 

Ok, now the hardest part. Please squash your pull request and rebase it on master. Also add your PR to the [release_notes.md](https://github.com/MehdiK/Humanizer/blob/master/release_notes.md)

You can find more info in [CONTRIBUTING.md](https://github.com/MehdiK/Humanizer/blob/master/CONTRIBUTING.md)

**Comment by dmytro-ielkin** [comment]
Ok. I was wrong to work on this PR in master. Will remove the repository and do it the right way.
